docs/conf/na/2017/writing-day.rst

```diff
@@ -28,3 +28,52 @@ Documenting a new project?
 Check out our `beginners guide <http://docs.writethedocs.org/writing/beginners-guide-to-docs/>`_ to writing documentation.
 This should help you get started,
 and give you some ideas for how you can contribute to a project that you love.
+
+Write Write the Docs on Writing Day
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This year we're also running a session where you can help improve your favourite website. Yup, you can brainstorm improvements, write helpful content and posts or just magically improve `Write the Docs <http://www.writethedocs.org>`_.
+
```

Comment:
"write helpful content and posts, ..."
(oxford comma. Buncha nits here because it's about a docs site :D)